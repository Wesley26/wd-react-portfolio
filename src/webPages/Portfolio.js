import React from 'react'
import { portfolioTextWeb1, portfolioTextPersonal1 } from './contentMaster/PageContents.js'

import ProjectOne from './portfolioComponents/projectOne.js'
import ProjectTwo from './portfolioComponents/projectTwo.js'
import ProjectThree from './portfolioComponents/projectThree.js'
import WebsiteOne from './portfolioComponents/websiteOne.js'

function Portfolio() {

    /**
     * Note: On JSX div elements line 22 and 32, for every 3 portfolio items added, the CSS class
     * 'lg:grid-rows-1' must increase by one row.
     */

    return (
        <div className="bg-gray-100 m-6">
            <div className="text-left p-6">
                <div className="flex justify-evenly">

                    <div className="left-0 p-3">
                        <h3 className="p-3 text-center font-bold">
                            {portfolioTextWeb1.titleP1}
                        </h3>

                        <div className="grid md:grid-cols-1 md:p-3 lg:grid-cols-3 lg:grid-rows-1 gap-4 justify-items-auto">
                            <div className="bg-gray-300 flex justify-center items-center h-auto w-auto">
                                <WebsiteOne />
                            </div>
                        </div>

                        <h3 className="p-3 text-center font-bold">
                            {portfolioTextPersonal1.titleP2}
                        </h3>

                        <div className="grid md:grid-cols-1 md:p-3 lg:grid-cols-3 lg:grid-rows-1 gap-4 justify-items-auto">
                            <div className="bg-gray-300 flex justify-center items-center h-auto w-auto">
                                <ProjectOne />
                            </div>
                            <div className="bg-gray-500 flex justify-center items-center h-auto w-auto">
                                <ProjectTwo />
                            </div>
                            <div className="bg-gray-300 flex justify-center items-center h-auto w-auto">
                                <ProjectThree />
                            </div>
                        </div>
                        
                    </div>

                </div>
            </div>
        </div>
    )
};

export default Portfolio