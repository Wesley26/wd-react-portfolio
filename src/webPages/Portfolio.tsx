import { FC, ReactElement, useEffect, useMemo, useRef, useState } from 'react';
import PortfolioCard from './portfolioComponents/PortfolioCard';
import PortfolioHeader from './portfolioComponents/PortfolioHeader';
import PortfolioNav from './portfolioComponents/PortfolioNav';
import MainPageAnimations from './contentMaster/MainPageAnimations';
import { motion } from 'framer-motion';

import { GitHubLinkedInURL } from './reusableComponents/reusableIndex';
import { PortfolioDisplayContext } from '../hooks/PortfolioDisplayContext';

import {
    MobileOne,
    MobileTwo,
    ProjectOne,
    RealWorldOne,
    WebsiteOne,
    WebsiteTwo,
    WebsiteThree,
} from './portfolioComponents/portfolioChildren/index';

/**
 * @param portfolioChildList - list all React Element
 * portfolio items in this Array as portfolio page
 * navigation is determined by its length.
 * @returns Portfolio page component
 */

const Portfolio:FC = () => {

    const mainPageAnimations = MainPageAnimations();
    const [visible, setVisible] = useState<boolean>(false);

    useEffect(() => {

        setVisible(false);
        setVisible(true);

    }, []);

    const [portfolioSetter, setPortfolioSetter] = useState<number>(0);

    const portfolioChildListMemo = useMemo(() => {

        const portfolioChildList: Array<ReactElement> = [
            /**
             * List each portfolio item component in this Array
             */
            ///////////////REAL WORLD PROJECTS/////////////////////////////
            <RealWorldOne />, //0, (Sketch/Eastman Core Mobile App) 
            ///////////////MOBILE APP PROJECTS/////////////////////////////
            <MobileOne />, //1, MobileOne (Weather Utility App)
            <MobileTwo />, //2, MobileTwo (CHOYCES Mobile)
            ///////////////WEBSITE PROJECTS////////////////////////////////
            <WebsiteOne />, //3, WebsiteOne, (Smile Design Website)
            <WebsiteTwo />, //4, WebsiteTwo, (Bit Broker Challenge)
            <WebsiteThree />, //5, WebsiteThree, (wd_vize Challenge)
            ///////////////PERSONAL PROJECTS///////////////////////////////
            <ProjectOne />, //6, ProjectOne, (Karters Discord Bot Project)
        ];

        return portfolioChildList;

    }, []);

    const portfolioNavRef = useRef<HTMLDivElement>(null);

    return (
        <div className="font-body bg-body_lightGray m-6 overflow-auto shadow-xl">

            <> { visible ? (
                <motion.div
                    initial={mainPageAnimations.initialStyle}
                    animate={mainPageAnimations.animateStyle}
                    transition={mainPageAnimations.transitionStyle}
                    className="text-left p-6"
                >
                    <GitHubLinkedInURL />
                    <div className="flex justify-evenly">

                        <div className="left-0 p-3">
                            
                            <PortfolioDisplayContext.Provider value={{ portfolioSetter, setPortfolioSetter, portfolioChildListMemo }}>

                                <PortfolioHeader />

                                <div ref={portfolioNavRef} />

                                <div className="flex flex-row justify-center items-center">

                                    <PortfolioNav scrollRef={portfolioNavRef} />

                                </div>

                                <div className="py-3" />

                                <div className="flex justify-center items-center bg-gray-300 border-2 rounded-md border-gray-800 shadow-xl">

                                    <PortfolioCard />

                                </div>

                            </ PortfolioDisplayContext.Provider>
                            
                        </div>

                    </div>
                </motion.div>
                ) : null }
            </>
        </div>
    )
};

export default Portfolio;