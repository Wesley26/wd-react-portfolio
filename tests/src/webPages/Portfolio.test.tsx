import { render } from '@testing-library/react';
import Portfolio from '../../../src/webPages/Portfolio';
import '../../src/index.css';

it(`render ContactMe.tsx test component`, () => {

    const view = (render(<Portfolio />));

    expect(view).toMatchSnapshot();

});