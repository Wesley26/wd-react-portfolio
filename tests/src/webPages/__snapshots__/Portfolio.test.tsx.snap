// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ContactMe.tsx test component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="font-body bg-body_lightGray m-6 overflow-auto shadow-xl"
      >
         
        <div
          class="text-left p-6"
          style="opacity: 0; transform: translateX(0px) translateY(-20px) translateZ(0);"
        >
          <div
            class="flex flex-row justify-between items-center ml-12 mr-12 md:ml-36 md:mr-36 lg:ml-72 lg:mr-72"
          >
            <div
              class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
            >
              <a
                href="https://github.com/Wesley26"
                rel="noreferrer"
                target="_blank"
              >
                <img
                  alt="Image has failed to load."
                  class="focus:outline-none"
                  src="[object Object]"
                />
              </a>
            </div>
            <div
              class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
            >
              <a
                href="https://www.linkedin.com/in/wesleydzitzer/"
                rel="noreferrer"
                target="_blank"
              >
                <img
                  alt="Image has failed to load."
                  class="focus:outline-none"
                  src="[object Object]"
                />
              </a>
            </div>
          </div>
          <div
            class="flex justify-evenly"
          >
            <div
              class="left-0 p-3"
            >
              <div
                class="text-lg p-6 text-center"
              >
                <p>
                  To view the entire design for this website, visit
                  <a
                    class="border-gray-700 border-2 p-3 rounded-full flex justify-center md:ml-48 md:mr-48 lg:ml-20long lg:mr-20long hover:bg-gray-200 active:bg-gray-400 shadow-md"
                    href="https://github.com/Wesley26/wd-react-portfolio"
                    rel="noreferrer"
                    target="_blank"
                  >
                    this following repo.
                  </a>
                </p>
                <p
                  class="pt-3"
                >
                  Use the navigation arrows below to view each portfolio item.
                </p>
                <p
                  class="pb-3"
                >
                  A project with a publicly viewable GitHub repository includes a 
        clickable link to the corresponding repository.
                </p>
                <p
                  class="pb-3"
                >
                  A project that does not contain a public repository includes a
        screenshot gallery and description.
                </p>
              </div>
              <div />
              <div
                class="flex flex-row justify-center items-center"
              >
                <div
                  class="p-3 flex rounded-full justify-center hover:bg-gray-400 active:bg-gray-500"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-caret-left text-9xl text-gray-100"
                    data-icon="caret-left"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    viewBox="0 0 256 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M9.4 278.6c-12.5-12.5-12.5-32.8 0-45.3l128-128c9.2-9.2 22.9-11.9 34.9-6.9s19.8 16.6 19.8 29.6l0 256c0 12.9-7.8 24.6-19.8 29.6s-25.7 2.2-34.9-6.9l-128-128z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                <div
                  class="px-12 md:px-44 lg:px-64"
                />
                <div
                  class="p-3 flex rounded-full justify-center hover:bg-gray-400 active:bg-gray-500"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-caret-right text-9xl text-gray-600"
                    data-icon="caret-right"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    viewBox="0 0 256 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
              <div
                class="py-3"
              />
              <div
                class="flex justify-center items-center bg-gray-300 border-2 rounded-md border-gray-800 shadow-xl"
              >
                <div
                  class="px-3 py-12 flex"
                >
                   
                   
                  <div
                    class="mx-1 flex flex-col justify-center items-center w-max"
                    style="opacity: 0; transform: translateX(0px) translateY(-10px) translateZ(0);"
                  >
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="p-3 flex rounded-full justify-center"
                      >
                        <img
                          alt="Image has failed to load."
                          class="focus:outline-none"
                          src="[object Object]"
                        />
                      </div>
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long md:w-96"
                      >
                        React Native CLI Mobile App
                      </p>
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long md:w-96"
                      >
                        Sketch Development/Eastman Chemical Company Mobile App
                      </p>
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long md:w-96"
                      >
                        All screens shown below were built by me from a Figma design.
        The mobile app started as a vanilla JavaScript React Native app. The mobile app received
        a TypeScript upgrade in the last week of its development period. This React Native mobile
        app, compatible for both iOS and Android, is designed for internal use with one of
        Eastman Chemical Company's vehicle trim vendor clients.
                      </p>
                    </div>
                    <div
                      class="py-3 bg-transparent"
                    />
                    <div>
                      <div
                        class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                      >
                        <div
                          class="bg-realWorld_oneA bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                        />
                        <div
                          class="py-3 bg-transparent"
                        />
                        <p
                          class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                        >
                          Android: No vehicle VIN ID found page
                        </p>
                      </div>
                      <div
                        class="pt-3 pb-6 bg-transparent"
                      />
                    </div>
                    <div>
                      <div
                        class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                      >
                        <div
                          class="bg-realWorld_oneB bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                        />
                        <div
                          class="py-3 bg-transparent"
                        />
                        <p
                          class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                        >
                          iOS: No vehicle VIN ID found page
                        </p>
                      </div>
                      <div
                        class="pt-3 pb-6 bg-transparent"
                      />
                    </div>
                    <div>
                      <div
                        class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                      >
                        <div
                          class="bg-realWorld_oneC bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                        />
                        <div
                          class="py-3 bg-transparent"
                        />
                        <p
                          class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                        >
                          Android: Vehicle Lookup selection screen
                        </p>
                      </div>
                      <div
                        class="pt-3 pb-6 bg-transparent"
                      />
                    </div>
                    <div>
                      <div
                        class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                      >
                        <div
                          class="bg-realWorld_oneD bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                        />
                        <div
                          class="py-3 bg-transparent"
                        />
                        <p
                          class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                        >
                          iOS: Vehicle Lookup selection screen
                        </p>
                      </div>
                      <div
                        class="pt-3 pb-6 bg-transparent"
                      />
                    </div>
                    <div>
                      <div
                        class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                      >
                        <div
                          class="bg-realWorld_oneE bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                        />
                        <div
                          class="py-3 bg-transparent"
                        />
                        <p
                          class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                        >
                          iOS: My Shop Notes empty state page
                        </p>
                      </div>
                      <div
                        class="pt-3 pb-6 bg-transparent"
                      />
                    </div>
                    <div>
                      <div
                        class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                      >
                        <div
                          class="bg-realWorld_oneF bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                        />
                        <div
                          class="py-3 bg-transparent"
                        />
                        <p
                          class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                        >
                          iOS: Select Model page with PPF/Tint options
                        </p>
                      </div>
                      <div
                        class="pt-3 pb-6 bg-transparent"
                      />
                    </div>
                  </div>
                   
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="font-body bg-body_lightGray m-6 overflow-auto shadow-xl"
    >
       
      <div
        class="text-left p-6"
        style="opacity: 0; transform: translateX(0px) translateY(-20px) translateZ(0);"
      >
        <div
          class="flex flex-row justify-between items-center ml-12 mr-12 md:ml-36 md:mr-36 lg:ml-72 lg:mr-72"
        >
          <div
            class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
          >
            <a
              href="https://github.com/Wesley26"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="Image has failed to load."
                class="focus:outline-none"
                src="[object Object]"
              />
            </a>
          </div>
          <div
            class="border-gray-700 border-2 p-3 flex rounded-full justify-center shadow-md hover:bg-gray-200 active:bg-gray-400"
          >
            <a
              href="https://www.linkedin.com/in/wesleydzitzer/"
              rel="noreferrer"
              target="_blank"
            >
              <img
                alt="Image has failed to load."
                class="focus:outline-none"
                src="[object Object]"
              />
            </a>
          </div>
        </div>
        <div
          class="flex justify-evenly"
        >
          <div
            class="left-0 p-3"
          >
            <div
              class="text-lg p-6 text-center"
            >
              <p>
                To view the entire design for this website, visit
                <a
                  class="border-gray-700 border-2 p-3 rounded-full flex justify-center md:ml-48 md:mr-48 lg:ml-20long lg:mr-20long hover:bg-gray-200 active:bg-gray-400 shadow-md"
                  href="https://github.com/Wesley26/wd-react-portfolio"
                  rel="noreferrer"
                  target="_blank"
                >
                  this following repo.
                </a>
              </p>
              <p
                class="pt-3"
              >
                Use the navigation arrows below to view each portfolio item.
              </p>
              <p
                class="pb-3"
              >
                A project with a publicly viewable GitHub repository includes a 
        clickable link to the corresponding repository.
              </p>
              <p
                class="pb-3"
              >
                A project that does not contain a public repository includes a
        screenshot gallery and description.
              </p>
            </div>
            <div />
            <div
              class="flex flex-row justify-center items-center"
            >
              <div
                class="p-3 flex rounded-full justify-center hover:bg-gray-400 active:bg-gray-500"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-caret-left text-9xl text-gray-100"
                  data-icon="caret-left"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 256 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M9.4 278.6c-12.5-12.5-12.5-32.8 0-45.3l128-128c9.2-9.2 22.9-11.9 34.9-6.9s19.8 16.6 19.8 29.6l0 256c0 12.9-7.8 24.6-19.8 29.6s-25.7 2.2-34.9-6.9l-128-128z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                class="px-12 md:px-44 lg:px-64"
              />
              <div
                class="p-3 flex rounded-full justify-center hover:bg-gray-400 active:bg-gray-500"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-caret-right text-9xl text-gray-600"
                  data-icon="caret-right"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 256 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
            <div
              class="py-3"
            />
            <div
              class="flex justify-center items-center bg-gray-300 border-2 rounded-md border-gray-800 shadow-xl"
            >
              <div
                class="px-3 py-12 flex"
              >
                 
                 
                <div
                  class="mx-1 flex flex-col justify-center items-center w-max"
                  style="opacity: 0; transform: translateX(0px) translateY(-10px) translateZ(0);"
                >
                  <div
                    class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                  >
                    <div
                      class="p-3 flex rounded-full justify-center"
                    >
                      <img
                        alt="Image has failed to load."
                        class="focus:outline-none"
                        src="[object Object]"
                      />
                    </div>
                    <p
                      class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long md:w-96"
                    >
                      React Native CLI Mobile App
                    </p>
                    <p
                      class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long md:w-96"
                    >
                      Sketch Development/Eastman Chemical Company Mobile App
                    </p>
                    <p
                      class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long md:w-96"
                    >
                      All screens shown below were built by me from a Figma design.
        The mobile app started as a vanilla JavaScript React Native app. The mobile app received
        a TypeScript upgrade in the last week of its development period. This React Native mobile
        app, compatible for both iOS and Android, is designed for internal use with one of
        Eastman Chemical Company's vehicle trim vendor clients.
                    </p>
                  </div>
                  <div
                    class="py-3 bg-transparent"
                  />
                  <div>
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="bg-realWorld_oneA bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                      />
                      <div
                        class="py-3 bg-transparent"
                      />
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                      >
                        Android: No vehicle VIN ID found page
                      </p>
                    </div>
                    <div
                      class="pt-3 pb-6 bg-transparent"
                    />
                  </div>
                  <div>
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="bg-realWorld_oneB bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                      />
                      <div
                        class="py-3 bg-transparent"
                      />
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                      >
                        iOS: No vehicle VIN ID found page
                      </p>
                    </div>
                    <div
                      class="pt-3 pb-6 bg-transparent"
                    />
                  </div>
                  <div>
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="bg-realWorld_oneC bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                      />
                      <div
                        class="py-3 bg-transparent"
                      />
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                      >
                        Android: Vehicle Lookup selection screen
                      </p>
                    </div>
                    <div
                      class="pt-3 pb-6 bg-transparent"
                    />
                  </div>
                  <div>
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="bg-realWorld_oneD bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                      />
                      <div
                        class="py-3 bg-transparent"
                      />
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                      >
                        iOS: Vehicle Lookup selection screen
                      </p>
                    </div>
                    <div
                      class="pt-3 pb-6 bg-transparent"
                    />
                  </div>
                  <div>
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="bg-realWorld_oneE bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                      />
                      <div
                        class="py-3 bg-transparent"
                      />
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                      >
                        iOS: My Shop Notes empty state page
                      </p>
                    </div>
                    <div
                      class="pt-3 pb-6 bg-transparent"
                    />
                  </div>
                  <div>
                    <div
                      class="p-3 flex flex-col justify-center items-center border-2 rounded-md border-gray-600 bg-blue-100 shadow-xl"
                    >
                      <div
                        class="bg-realWorld_oneF bg-cover bg-no-repeat border-2 rounded-md border-gray-600 h-30long md:h-51.6long w-56 md:w-96"
                      />
                      <div
                        class="py-3 bg-transparent"
                      />
                      <p
                        class="p-1 font-title bg-gray-500 text-gray-200 text-center font-medium w-11long"
                      >
                        iOS: Select Model page with PPF/Tint options
                      </p>
                    </div>
                    <div
                      class="pt-3 pb-6 bg-transparent"
                    />
                  </div>
                </div>
                 
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
